registrations/models.py

```diff
@@ -73,6 +78,26 @@ class Registration(models.Model):
     def __str__(self):
         return str(self.id)
 
+    def get_voice_days_and_times(self):
+        return self.data.get('voice_days'), self.data.get('voice_times')
+
+    def get_preg_week(self):
+        week = self.data.get("preg_week") or self.data.get("baby_age")
+        if week is not None:
+            return week
+        raise RegistrationException(
+            'Neither preg_week or baby_age are specified for %s.' % (self,))
+
+    def get_subscription_requests(self):
+        return SubscriptionRequest.objects.filter(identity=self.mother_id)
```

Comment:
Ties in with my comment in `commands/verify_registration_schedule` - we can discuss in a call if that doesn't clarify enough.